CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
   The new architecture also allows parallel I/O writing and storing of multiple job streams in one stream handle.
   You can use worker streaming using the `--stream` parameter of `hq submit`. Check out the documentation for more information.
 
+* Optimization of journal size
```

Comment:
Maybe we could mention/remind that dashboard is still disabled.

Reply:
It is a status quo, and this should be a changelog, isnt it?:)

Reply:
I know, I know :) But some people might have expected that the next version will enable it again.

Reply:
Ok, I am adding a note